bin/acala-dev/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "acala-dev"
+version = "0.7.0"
+authors = ["Acala Developers"]
+edition = "2018"
+
+[[bin]]
+name = "acala-dev"
+path = "src/main.rs"
+
+[dependencies]
+futures = "0.3.4"
+parity-util-mem = { version = "0.7.0", default-features = false, features = ["jemalloc-global"] }
+acala-dev-cli = { path = "cli" }
+acala-dev-service = { path = "service" }
+mandala-runtime = { path = "../../runtime/mandala", optional = true }
+
+[features]
+default = [ "with-mandala-runtime" ]
+runtime-benchmarks = [
+	"acala-dev-cli/runtime-benchmarks",
+]
+with-mandala-runtime = [
+	"acala-dev-service/with-mandala-runtime",
+	"acala-dev-cli/with-mandala-runtime",
+]
+with-all-runtime = [
```

Comment:
no need to have `with-xxx-runtime` feature for dev. this should only be one supported runtime